<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifier_visite</class>
 <widget class="QDialog" name="modifier_visite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modfier visite</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(29, 30, 48);</string>
  </property>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;Pr√®nom patient&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_naiss_patient">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
background-color:#25283b;

border:1px solid rgb(61, 70, 86);
color:rgb(255, 255, 255);
padding-left:16px;
border:1px solid rgb(61, 70, 86);
border-top-right-radius:10px;
border-down-right-radius:10px;
}
QDateEdit:focus{
border: 1px solid rgb(180, 180, 180);
}</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nom_patient_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color:#25283b;

border:1px solid rgb(61, 70, 86);
color:rgb(255, 255, 255);
padding-left:16px;
border:1px solid rgb(61, 70, 86);
border-top-right-radius:10px;
border-down-right-radius:10px;
}
QLineEdit:focus{
border: 1px solid rgb(180, 180, 180);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenom_patient">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color:#25283b;

border:1px solid rgb(61, 70, 86);
color:rgb(255, 255, 255);
padding-left:16px;
border:1px solid rgb(61, 70, 86);
border-top-right-radius:10px;
border-down-right-radius:10px;
}
QLineEdit:focus{
border: 1px solid rgb(180, 180, 180);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;Date naissance du patient&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;Diagnostic&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="diagnostic">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>570</y>
     <width>621</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
background-color:#25283b;

border:1px solid rgb(61, 70, 86);
color:rgb(255, 255, 255);
padding-left:16px;
border:1px solid rgb(61, 70, 86);
border-top-right-radius:10px;
border-down-right-radius:10px;
}
QTextEdit:focus{
border: 1px solid rgb(180, 180, 180);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;Nom patient&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;Observation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="observation">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>621</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
background-color:#25283b;

border:1px solid rgb(61, 70, 86);
color:rgb(255, 255, 255);
padding-left:16px;
border:1px solid rgb(61, 70, 86);
border-top-right-radius:10px;
border-down-right-radius:10px;
}
QTextEdit:focus{
border: 1px solid rgb(180, 180, 180);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; text-decoration: underline; color:#ffffff;&quot;&gt;Modifier Visite&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>750</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {	
border:none;
background-color:#0000FF;
	background-position: left center;

    background-repeat: no-repeat;

	border-radius: 5px;
border:1px solid transparent;
color:rgb(255, 255, 255);
	text-align: center;

border-left:4px solid #0000FF;
	
}
QPushButton:hover {
	background-color:rgb(0, 0, 128);
	
}
QPushButton:pressed {
border:2px solid rgb(0, 0, 128);


}
</string>
   </property>
   <property name="text">
    <string>Valider </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>750</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {	
border:none;
background-color:#0000FF;
	background-position: left center;

    background-repeat: no-repeat;

	border-radius: 5px;
border:1px solid transparent;
color:rgb(255, 255, 255);
	text-align: center;

border-left:4px solid #0000FF;
	
}
QPushButton:hover {
	background-color:rgb(0, 0, 128);
	
}
QPushButton:pressed {
border:2px solid rgb(0, 0, 128);


}
</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color:#25283b;

border:1px solid rgb(61, 70, 86);
color:rgb(255, 255, 255);
padding-left:16px;
border:1px solid rgb(61, 70, 86);
border-top-right-radius:10px;
border-down-right-radius:10px;
}
QLineEdit:focus{
border: 1px solid rgb(180, 180, 180);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;ID_Visite&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
